rpc/flipt/validation.go

```diff
@@ -420,6 +437,10 @@ func (req *CreateConstraintRequest) Validate() error {
 			return err
 		}
 		req.Value = v
+	} else if operator == "isoneof" {
```

Comment:
could use the `OpIsOneOf` constant from: https://github.com/flipt-io/flipt/pull/2368/files#diff-c85cd8cee3ae90043ef906cc0bbacaf09d014ffbb13fcc16d2216933deafe2e2R54